ntp.go

```diff
@@ -31,6 +31,10 @@ const (
 const (
 	maxStratum = 16
 	nanoPerSec = 1000000000
+	leapNoWarning = 0
+	leapAddSecond = 1
+	leapDelSecond = 2
+	leapNotInSync = 3
```

Comment:
I suggest to have these public constants as a part of "API" saying that Response.Leap is not {-1,0,1} but leapNoWarning / leapAddSecond / leapDelSecond / leapDelSecond. What can be better way to express that?